src/engines/html5/html5.js

```diff
@@ -182,6 +188,11 @@ export default class Html5 extends FakeEventTarget implements IEngine {
   attach(): void {
     Object.keys(Html5Events).forEach((html5Event) => {
       this._eventManager.listen(this._el, Html5Events[html5Event], () => {
+        if (Html5Events[html5Event] == Html5Events.PLAYING &&
+          !this._registeredMetadataEvent &&
+          this._config.playback.registerMetadataTrackEvent){
```

Comment:
@itaykinnrot  why we need this configurable?

Reply:
We want to enable it only if a plugin is requesting this info - since sometimes it can cause issues in browsers (safari :-( )

Reply:
@itaykinnrot if we need to do this let's move it to it's own event listener:
```javascript
this._eventManager.listen(this._el, Html5Events.PLAYING, () => this. _shouldAddMetadataEvent);
```

and move the logic if to add it or not to the handler itself